spec/database_spec.rb

```diff
@@ -8,29 +8,27 @@
   end
 
   describe "path" do
-    subject { described_class.path }
-
     it "it should be a directory" do
-      expect(File.directory?(subject)).to be_truthy
+      expect(described_class.path).to be_truthy
     end
 
-    it "should prefer the user repo, iff it's as up to date, or more up to date than the vendored one" do
+    it "should prefer the user repo, iff it's as up to date, or more up to date than the vendored one", focus: true do
```

Comment:
This should probably not have `focus: true`.

Reply:
Thank you for catching that, I've removed it.